src/net/sf/memoranda/ui/AppFrame.java

```diff
@@ -689,8 +689,7 @@ protected void processWindowEvent(WindowEvent e) {
         }
         else if ((e.getID() == WindowEvent.WINDOW_ICONIFIED)) {
             super.processWindowEvent(new WindowEvent(this,
-                    WindowEvent.WINDOW_CLOSING));
-            doMinimize();
+                    WindowEvent.WINDOW_ICONIFIED));
```

Comment:
Now I am wondering what the original intent of the code was. It is almost as though that author intended to close the window on minimizing. After all, how is this different from from letting this case fall through to line 695?